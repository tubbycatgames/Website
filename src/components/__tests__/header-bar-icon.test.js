import React from "react"
import renderer from "react-test-renderer"

import HeaderBarIcon from "../header-bar-icon"

describe("Header Bar Icon", () => {
  it("renders a header bar icon", () => {
    const tree = renderer.create(
      <HeaderBarIcon
        alt="Some Alt Text!"
        href="Some madeup place"
        src="some-image.jpg"
        style="A style override!"
      />
    )
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
