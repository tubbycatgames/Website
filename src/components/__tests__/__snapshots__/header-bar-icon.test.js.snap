// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Bar Icon renders a header bar icon 1`] = `
<a
  href="Some madeup place"
>
  <img
    alt="Some Alt Text!"
    className="A style override!"
    src="some-image.jpg"
  />
</a>
`;
